### ISSUE TITLE
LUIS Gen build (via AzureDevOps) is broken. 

### ISSUE THREAD

**Issue by cleemullins** [Microsoft/botbuilder-tools#653]
The LuisGen build/signing seems to be broken. It’s been broken for about a week. 

```
2018-10-11T09:00:33.3861363Z [command]C:\Windows\system32\chcp.com 65001
2018-10-11T09:00:33.4579360Z Active code page: 65001
2018-10-11T09:00:33.4779382Z ##[error]Project file(s) matching the specified pattern were not found.
2018-10-11T09:00:33.4871748Z ##[section]Finishing: dotnet build LUISGen

```
There are 3 levels of things that are wrong: 

1.	Bad Path to LuisGEN.csproj in the Azure DevOps build and pack steps
	[Existing] Bad Path: packages/LUISGen/src/LUISGen/LUISGen.csproj
	[Revised] Right Path:  packages/LUISGen/src/LUISGen.csproj

2.	Incorrect signing key path: 
```
2018-10-11T23:32:18.0398448Z CSC : error CS7027: Error signing output with public key from file '..\..\..\..\build\35MSSharedLib1024.snk' -- File not found. [D:\a\1\s\packages\LUISGen\src\LUISGen.csproj]
2018-10-11T23:32:18.0399229Z CSC : warning CS7033: Delay signing was specified and requires a public key, but no public key was specified [D:\a\1\s\packages\LUISGen\src\LUISGen.csproj]
2018-10-11T23:32:18.0476962Z 
2018-10-11T23:32:18.0477464Z Build FAILED.
```

The SNK file is defined in the .csproj:
        <AssemblyOriginatorKeyFile>..\..\..\..\build\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>

There is one extra “..\”.  I fixed this (PR here).

In short, the build + sign + publish of LuisGen works again.
